src/Domain/Card/Deck/AnswerDeckInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Nusje2000\CAH\Domain\Card\Deck;
+
+use Nusje2000\CAH\Domain\Card\AnswerInterface;
+
+interface AnswerDeckInterface
```

Comment:
why not create a `DeckInterface` instead of splitting it between `Question` and `Answer`?